pkg/kubeconfig/kubeconfig.go

```diff
@@ -115,11 +117,11 @@ func ForTokenBased(clusterName, clusterID, host, username, token string) (string
 func ForClusterTokenBased(cluster *managementv3.Cluster, clusterID, host, username, token string) (string, error) {
 	nodes := []node{getDefaultNode(cluster.Name, clusterID, host, username)}
 
-	if cluster.ClusterEndpointFQDN != "" {
+	if cluster.LocalClusterAuthEndpoint.FQDN != "" {
 		clusterNode := node{
 			ClusterName: cluster.Name + "-fqdn",
-			Server:      "https://" + cluster.ClusterEndpointFQDN + ":6443",
-			Cert:        formatCertString(cluster.ClusterEndpointFQDNCaCert),
+			Server:      "https://" + cluster.LocalClusterAuthEndpoint.FQDN + ":6443",
```

Comment:
oh, the expectation is that if the user is supplying the fqdn, port 6443 should not need to be supplied.

Reply:
either the user specifies the port or their lb gets traffic on 443 and forwards to 6443

Reply:
Fixed, thanks for finding this!